src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure {
+    private String color;
```

Comment:
let's create a constructor with parameter `color` and call this constructor in its subclasses to initialize field `color`